pkg/render/json.go

```diff
@@ -27,6 +27,8 @@ func (r JSON) File(_ context.Context, _ *malcontent.FileReport) error {
 }
 
 func (r JSON) Full(_ context.Context, rep *malcontent.Report) error {
+	// Drop the applied filters
```

Comment:
We don't display this for any other renderer (though it's still set in the report) and it was causing weird diffs when testing JSON output.